### ISSUE TITLE
[FEAT] Place pyradio-recordings by default in .config/pyradio (or don't create at all)

### ISSUE THREAD

**Issue by lee2sman** [coderholic/pyradio#253]
As described in #241 there is a default pyradio-recordings directory created when starting up pyradio (if it doesn't already exist). Creating a new directory in a user's home directory seems like a no-go to me, and messes up my backup system, among many other things.  I could be wrong, but i'd imagine most users don't even know about the recording feature, much less make use of it, so this seems "heavy-handed" of sorts.

Proposed solution: 

Ideally, pyradio shouldn't create a directory for pyradio-recordings until the first time a user tries to make a recording. Also, the folder should by default be inside the ~/.config/pyradio folder, not inside home.

Output of my pyradio --version command:

```
XDG Dirs found; enabling XDG Base compliant operation
PyRadio version: 0.9.3.9
Python version: 3.12.5 (main, Aug  7 2024, 19:14:22) [GCC 13.2.0]
Distribution: Void Linux
```

**Comment by s-n-g** [comment]
On it!

**Comment by s-n-g** [comment]
fixed in [0.9.3.10](https://github.com/coderholic/pyradio/releases/tag/0.9.3.10)
